src/manager/msWindowManager.ts

```diff
@@ -681,6 +685,15 @@ export class MsWindowManager extends MsManager {
                 msWindow.lifecycleState.type === 'app-placeholder' &&
                 msWindow.lifecycleState.waitingForAppSince !== undefined
             ) {
+                // If there is an authentication dialog postpone the window cleaning process
```

Comment:
Why do we have to do this?

Reply:
When you try to open an app that require `sudo` there is an system authentication dialog that ask for the sudo password. 
Our routine detect this has app didn't opened properly and close the `MsWindow` before the app could be opened which lead to actualizing the focus on another tileable stealing the focus from the authentication dialog

Reply:
Great! Add a comment :)